apps/api/src/database/seed/data-samples.js

```diff
@@ -429,6 +429,21 @@ export const represenations = [
 					phoneNumber: '01234 567890'
 				}
 			]
+		},
+		RepresentationActions: {
+			create: [
+				{
+					type: 'REDACT',
```

Comment:
I guess if it's a REDACT the  redactStatus and previousRedactStatus are important not really the status fields.
Again, sorry this details wasn't on the ticket.